Model timing update (#7476)
fishtown-analytics/docs.getdbt.com: website/docs/best-practices/materializations/materializations-guide-6-examining-builds.md

- 🧵 This view lets us see our **mapped out in threads** (up to 64 threads, we’re currently running with 4, so we get 4 tracks) over time. You can think of **each thread as a lane on a highway**.
- ⌛ We can see above that `order_items` and `orders` are **taking the most time**, so we may want to go ahead and **make that incremental**.
- :one: If a job has a single dbt invocation (for example `dbt build`), the model timing chart reflects the timing of all models.

> - :1234: If a job includes multiple dbt commands such as `dbt build` followed by `dbt compile` the model timing chart will reflect only the models from the final command `dbt compile`. For models executed in both commands, the chart will display the timing from the last invocation. Models that were not re-invoked in the final command will retain their timing from the earlier command `dbt build`.

Reviewer: ```suggestion
- :1234: If a job includes multiple dbt commands (for example, `dbt build` followed by `dbt compile`), the model timing chart reflects only the models from the final command (`dbt compile`). For models executed in both commands, the chart displays the timing from the last invocation. Models that were not re-invoked in the final command retain their timing from the earlier command (`dbt build`).
```
